### PR TITLE
Update to Yasson 1.0.11 and 3.0.2

### PR DESCRIPTION
**Pull request by njr-11**
Updates Liberty to Yasson 1.0.11 and 3.0.2 to stay current with any fixes that have been made.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -527,8 +527,9 @@ org.eclipse.transformer:org.eclipse.transformer.bnd.analyzer:0.5.0
 org.eclipse.transformer:org.eclipse.transformer.cli:0.5.0
 org.eclipse.transformer:org.eclipse.transformer.jakarta:0.5.0
 org.eclipse.transformer:org.eclipse.transformer:0.5.0
+org.eclipse:yasson:1.0.11
```
Comment (jhanders34) [inline dev/cnf/oss_dependencies.maven:530]: Should `org.eclipse:yasson:1.0.8` be removed from `cnf/oss_source_dependencies.maven`?

Comment (njr-11) [reply dev/cnf/oss_dependencies.maven:530]: Yes, I missed that file.  Thanks for spotting this.

**Comment by njr-11** [comment]
#libby
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by njr-11** [comment]
Build has 1 test failure which is a pre-existing issue in an unrelated area.

**Pull request merged**